src/hf_hydrodata/point.py

```diff
@@ -265,6 +281,12 @@ def get_point_metadata(*args, **kwargs):
         Latitude range bounds for the geographic domain; lesser value is provided first.
     longitude_range : tuple, optional
         Longitude range bounds for the geographic domain; lesser value is provided first.
+    grid : str, optional
```

Comment:
why are grid and grid_bounds added in the documentation for the get_point_data() and get_point_metadata() function?  It looks like those input variables are only made use of in the get_site_variables function?  I might be missing something or misunderstanding something here.

Reply:
This is a good question. These are used in get_point_data() and get_point_metadata() as passed in arguments that get further passed along as "options" to the internal _get_sites() function, which is called from each of those. The get_site_variables() function does the filtering a little differently, so it is not able to use the _get_sites() function and so in that function these fields get used a bit more explicitly.

Reply:
Got it, thanks for the explanation.